format_spec.md

```diff
@@ -0,0 +1,289 @@
+# MCIP Format Specification
+
+This document provides detailed information on the MCIP Format.
+
+The MCIP format uses JSON to store information. If providing an API that returns format-compliant information, the `application/json` Content-Type header SHOULD be provided. If stored on disk, the `.json` extension SHOULD be used.
+
+Metadata MAY be modularized in order to save bandwidth or storage space. Information on what fields may be modularized and the proper method of doing it will be at the end of this document.
+
+An format-compliant example is provided in `format_example.json`
+
+## Fields
+
+Fields are stored inside the JSON metadata of a Project. Fields that are optional will be marked as such. All fields are stored as a String value unless stated otherwise. Below is a complete list of all fields available for the MCIP Format, grouped into categories.
+
+## General Project Information
+
+### `schemaVersion`
+
+This field MUST be stored as an integer. It contains the version of the Format being used, to help with backwards compatibility.
+
+---
+
+### `id`
+
+The ID of the Project. There is no standard or required format on how to store an ID, other than it CANNOT include spaces. It may be a slug, UUID, random numer, or other method of storing IDs. It is RECOMMENDED that IDs are not tied to the name of a project.
+
+---
+
+### `name`
+
+The name of a Project. This name MUST be a human readable name and SHOULD not contain unnecessary information. For instance, the name field should not specify where the project is being hosted, or the authors of a project.
+
+---
+
+### `summary` (optional)
+
+A short, one sentence summary of the Project. HTML and Markdown SHOULD NOT be used.
+
+---
+
+### `icon` (optional)
+
+A URL to an icon of the Project. It is RECOMMENDED to use JPEG or PNG images. Images encoded into the field, such as a Base64 encoded PNG, SHOULD NOT be used.
+
+---
+
+## Detailed Project Information
+
+### `description` (optional)
+
+A long description of the project. HTML and Markdown MAY be used. Any form of code execution such as JavaScript or WebAssembly SHOULD NOT be used. HTML forms, buttons, or other form of interactive elements SHOULD NOT be used. iframes to external domains MAY be used, but it is recommended to embed iframes only with a trustworthy website.
```

Comment:
Markdown is bad for consistency. It will render incorrectly on every site other than the one it came from. How about XHTML?